handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+</figure>
+
+!!! note
+    Refer to [Setting up a SystemLink Server](https://www.ni.com/documentation/en/systemlink/2020r4/setup/setting-up-systemlink-server/) for information regarding ports used by SystemLink. d
+
+- **SystemLink App Server**:
+    - This Windows server hosts NI Web Server, the various SystemLink services, SystemLink web applications, and a Salt master.
+    - In the *single-box* setup, MongoDB and FIS storage are hosted on the app server.
+    - Refer to [Sizing a SystemLink Server](https://www.ni.com/documentation/en/systemlink/2020r4/setup/sizing-a-systemlink-server/) for details on server system requirements.
+    - NI assumes users with desktop access to the server are outside of the security controls and capabilities described by this document. Securing desktop access to the SystemLink app server should be managed by the customer's IT security protocols.
+
+- **SystemLink Web app**
+    - The SystemLink web application is the primary way users interact with SystemLink.
+    - NI recommends HTTPS for all production environments.
+    - Refer to [**NI Web Server**](#ni-web-server) for details on configuring encrypted communication.
+
+- **Targets**
+    - Targets are Windows and Linux RT systems who software and configuration are managed by SystemLink.
+    - Software configuration occurs via the [SaltStack Transport Protocol](https://docs.saltproject.io/en/latest/topics/transports/).
+        - Refer to [**Target to SystemLink Communication**](#target-to-systemlink-communication) for details regarding how SystemLink encrypts this communication.
+    - File, tag, test, and asset data are published to the SystemLink app server over HTTPS.
+
+- **MongoDB**
+    - MongoDB is the primary database used by SystemLink's web services.
+    - The [MongoDB wire protocol](https://docs.mongodb.com/manual/reference/mongodb-wire-protocol/) is used to communicate with the MongoDB instance.
+    - Refer to [TLS for Remote MongoDB Instances](#tls-for-remote-mongodb-instances) for details on enabling TLS communication.
+
+- **File Ingestion Service Storage**
+    - The File Ingestion Services (FIS) can be configured to store files on either Network Attached Storage (NAS) or AWS S3.
+    - NAS uses the SMB protocol.
+        - The SMB protocol can be configured to use AES encryption. Refer to [SMB security enhancements](https://docs.microsoft.com/en-us/windows-server/storage/file-server/smb-security) for details and steps to secure this transport.
+    - AWS S3 uses HTTPS.
+        Refer to the [SystemLink manual](https://www.ni.com/documentation/en/systemlink/latest/data/uploading-files-to-amazon-s3/) for steps to enable S3 file storage.
+
+- **Data Finder**
+    - DataFinder enables indexing and searching for files stored on network drives.
+    - DataFinder's network communication cannot be secured with TLS.
+
+- **Identity Provider**
+    - Active Directory, LDAP, and OpenID Connect are the supported identity providers.
```

Comment:
Consider omitting "the". Or something like "SystemLink supports Active Directory, LDAP, and OpenID Connect as identity providers."